apps/labs/next.config.js

```diff
@@ -30,18 +30,56 @@ const nextConfig = {
       },
     ];
   },
-  async rewrites() {
+  async redirects() {
     return [
       {
-        source: '/plausible/js/script.js',
-        destination: 'https://plausible.io/js/script.js',
-      },
-      {
-        source: '/plausible/api/event',
-        destination: 'https://plausible.io/api/event',
+        source: '/about',
+        destination: '/',
+        permanent: true,
       },
     ];
   },
+  async rewrites() {
+    // The conventions of this return value are described in the Next.js docs:
+    // https://nextjs.org/docs/api-reference/next.config.js/rewrites
+    return {
+      afterFiles: [
+        // These rewrites are checked after pages/public files are checked but
+        // before dynamic routes.
+
+        // Proxying Plausible through Vercel
+        // https://plausible.io/docs/proxy/guides/vercel
+        {
+          source: '/plausible/js/script.js',
+          destination: 'https://plausible.io/js/script.js',
+        },
+        {
+          source: '/plausible/api/event',
+          destination: 'https://plausible.io/api/event',
+        },
+      ],
+      fallback: [
+        // These rewrites are checked after both pages/public files and dynamic
+        // routes are checked.
+        {
+          source: '/:file(.+\\..+)',
```

Comment:
This is to catch the URL `/archive.html` (and any other like that) where Netlify does not expect a trailing slash.